### PR TITLE
Created the first repo settings with basic routes and the jira connector

### PR DESCRIPTION
**Pull request by lucasbald**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest"
+    },
+    "rules": {
+		"indent": [2, "tab", { "SwitchCase": 1, "VariableDeclarator": 1 }],
```
Comment (VitaoCs) [inline .eslintrc.json]: minor: wrong indentation

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "metrics-automation-reporter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (VitaoCs) [inline package.json]: We can include audit and lint checks to include on the pre-commit:
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
    "audit": "npm audit",
    "audit:fix": "npm audit fix",
    "lint:fix": "eslint . --fix",
    "lint": "eslint ."
  },
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "type": "git",
+    "url": "git+https://github.com/lucasbald/metrics-automation-reporter.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/lucasbald/metrics-automation-reporter/issues"
+  },
+  "homepage": "https://github.com/lucasbald/metrics-automation-reporter#readme",
```
Comment (VitaoCs) [inline package.json:23]: We can include engines and pre-commit:
```suggestion
  "homepage": "https://github.com/lucasbald/metrics-automation-reporter#readme",
  "engines": {
    "node": "16.x"
  },
  "pre-commit": [
	"audit",
	"lint"
  ],
```

File: .eslintrc.json (hunk 1 above)
Comment (lucasbald) [reply .eslintrc.json]: fixed

File: package.json (hunk 2 above)
Comment (lucasbald) [reply package.json]: added

File: package.json (hunk 3 above)
Comment (lucasbald) [reply package.json:23]: added precommit package as well
image

**Review by VitaoCs — APPROVED** [review]

**Pull request merged**